.github/workflows/integration.yml

```diff
@@ -34,10 +36,26 @@ on:
   # manual trigger
   workflow_dispatch:
     inputs:
-      dbt-core-branch:
+      dbt_core_branch:
         description: "branch of dbt-core to use in dev-requirements.txt"
         required: false
         type: string
+        default: main
+      name:
+        description: "name to associate with run"
+        required: false
+        type: string
+        default: "Adapter Integration Tests"
+      dbt_adapters_branch:
+        description: "The branch of dbt-adapters to use"
+        type: string
+        required: false
+        default: main
+      adapter_branch:
```

Comment:
I would call this `dbt_snowflake_branch` to align with `dbt_adapters_branch` and `dbt_core_branch`. We already need to know we're calling a workflow in the `dbt-snowflake` repo, so we don't gain much from making it generic.

Reply:
The exact same call is going to be made to each repo, so giving it the same field name DOES simplify it a lot. If every repo has a different field name for the local branch, then a different call has to be constructed for every repo.